source/package.json

```diff
@@ -1,13 +1,19 @@
 {
-  "dependencies": {
-    "jest": "^29.2.2",
-    "jest-environment-jsdom": "^29.2.2"
-  },
-  "scripts": {
-    "test": "jest --testEnvironment='jsdom'",
-    "lint": "eslint ."
-  },
-  "devDependencies": {
-    "eslint": "^8.26.0"
-  }
+    "dependencies": {
+        "jest": "^29.2.2",
+        "jest-environment-jsdom": "^29.2.2"
+    },
+    "scripts": {
+        "test": "jest --testEnvironment='jsdom'",
+        "lint": "eslint . && prettier --check ."
```

Comment:
This should actually do formatting. remove the --check

Reply:
oh I see how you did it in the readme. Then add another format script to do formatting. I don't know why the prettier is called using npx, we should be able to do NPM for everything.